coalib/collecting/Importers.py

```diff
@@ -15,8 +15,13 @@ def _import_module(file_path):
     module_name = os.path.splitext(os.path.basename(file_path))[0]
     module_dir = os.path.dirname(file_path)
 
+    # Force insert `module_dir` at front of the `sys.path`
     if module_dir not in sys.path:
         sys.path.insert(0, module_dir)
+    elif sys.path.index(module_dir) == 0:
+        pass
+    else:
+        sys.path.insert(0, sys.path.pop(sys.path.index(module_dir)))
```

Comment:
Need a warning if `module_dir` was in `sys.path`, because it is going to be removed on line 40, and the user probably either 1) was manipulating sys.path to workaround this bug (so they can now drop their workaround code), or 2) has some other reason and they are going to be surprised that we're deleting it from `sys.path`

Reply:
okay